src/components/UserManagement/UserManagement.jsx

```diff
@@ -325,6 +326,21 @@ class UserManagement extends React.PureComponent {
    * Callback to trigger on the status (active/inactive) column click to show the confirmaton change the status
    */
   onActiveInactiveClick = user => {
+    const authRole = this.props.state.auth.user.role;
+    const userPermissions = this.props.state.auth.user?.permissions?.frontPermisssion;
+    const { roles } = this.props.state.role;
+    const canChangeUserStatus = hasPermission(authRole, 'changeUserStatus', roles, userPermissions);
+    if (!canChangeUserStatus) {
+      //permission to change the status of any user on the user profile page or User Management Page.
+      //By default only Admin and Owner can access the user management page and they have this permission.
+      alert('You are not authorized to change the active status.');
+      return;
+    }
+    if (user.role === 'Owner' && user.isActive && authRole !== 'Owner') {
```

Comment:
Can this validation occur in another table/form?? maybe it's a good idea to create a function for this.